cli/src/lib.rs

```diff
@@ -1,3 +1,4 @@
+#![cfg_attr(coverage_nightly, feature(no_coverage))]
```

Comment:
is this a mistake? in other places you use `#[cfg_attr(coverage_nightly,no_coverage)]`

Reply:
nvm, I see the docs on `cargo-llvm-cov`

Reply:
Seems like that in `cargo-llvm-cov` but I'm confused by this

Reply:
So the point is we need to enable this nightly feature conditional on running coverage via nightly. 

The remaining places just mark functions as `no_coverage`.